src/domain/users/index.types.d.ts

```diff
@@ -39,14 +66,22 @@ type User = {
   readonly defaultAccountId: AccountId
   readonly deviceTokens: DeviceToken[]
   readonly createdAt: Date
-  phone: PhoneNumber
+  readonly phone: PhoneNumber
+  readonly phoneMetadata: PhoneMetadata | null
   language: UserLanguage
   twoFA: TwoFAForUser
 }
 
+type NewUserInfo = {
+  phone: PhoneNumber
+  phoneMetadata?: PhoneMetadata
```

Comment:
Should be consistently `| null`. In general we are avoiding `?` arguments as they are typically an indication of multiple use-cases being merged into one. Legitimate reasons for `absence` should be codified via `null`.